python/views.py

```
@@ -22,8 +22,7 @@
 
 import orjson
 import transformers
-from api import article, audio, comment, homepage, live, live_area, opus, user, video, video_zone
-from bilibili_api import search
+from api import article, audio, comment, homepage, live, live_area, opus, search, user, video, video_zone
```

Ruff (0.16.3)

[error] Module level import not at top of file

(E402)